lang/en/tool_userupsert.php

```diff
@@ -37,3 +37,13 @@
 authType | Auth type of the user.
 </pre>';
 $string['userupsert:upsert'] = 'Upsert users';
+$string['datamapping'] = 'Use the following options to map data from Web service fields to user profile fields. Any data not mapped to a profile field will be ignored.';
+$string['usermatchfield'] = 'User match field';
+$string['usermatchfield_desc'] = 'This field will be used for finding a user in Moodle. Make sure that you have mapped this it with Web service fields in Data mapping.';
+$string['notconfigured'] = 'The plugin is not configured properly. Please check the errors below.';
+$string['mappingerror'] = 'Web service field mapping is required for this field.';
+$string['defaultauth'] = 'Default authentication method';
+$string['auth'] = 'Authentication method';
+$string['defaultauth_desc'] = 'If an authentication method is not provided in Web service data, then this auth method will be set by default.';
+$string['status'] = 'Status - active, deleted or suspended';
+$string['error:morethanoneuser'] = 'More than one user found.';
```

Comment:
I know you don't like comments like this, but having lang strings sorted alphabetically makes the file neat and easy to work with (eg when looking for a string or deciding where to insert a new string).

Reply:
I will consider sorting strings as a final commit before claiming the plugin to be completed.